front/pages/api/w/[wId]/members/[uId]/index.ts

```diff
@@ -25,10 +22,14 @@ async function handler(
   res: NextApiResponse<WithAPIErrorResponse<PostMemberResponseBody>>,
   auth: Authenticator
 ): Promise<void> {
+  const owner = auth.getNonNullableWorkspace();
+
   if (!auth.isAdmin()) {
     // Allow Dust Super User to force role for testing
     const allowForTesting =
-      isDevelopment() && auth.isDustSuperUser() && req.body.force === "true";
+      isDevelopmentOrDustWorkspace(owner) &&
```

Comment:
Here we don't need to hard code the workspace right ?

Also isDevelopement is useful in itself for potential other use cases?

Reply:
I'm not sure to understand what you are saying, but will try to answer anyway.

> Here we don't need to hard code the workspace right ?

You mean only keep `auth.isDustSuperUser() && req.body.force === "true";` and remove the other conditions ?

> Also isDevelopement is useful in itself for potential other use cases?

I don't understand this one, seems contradictory with what you are saying above (isDevelopmentOrDustWorkspace() means either dev env or dust prod workspaces).

Reply:
> You mean only keep auth.isDustSuperUser() && req.body.force === "true"; and remove the other conditions ?

Yes

> I don't understand this one, seems contradictory with what you are saying above (isDevelopmentOrDustWorkspace() means either dev env or dust prod workspaces).

For this very feature yes but for other things we might want to use isDevelopment() right?

So let's have isDevelopment() and isDustWorkspace() and use || instead of intrudocing a new `isDevelopmentOrDustWorkspace` which will be an incentive to other developers to use this without thinking about the fact that this may impact our production

Reply:
isDevelopmentOrDustWorkspace is already there.

Reply:
Gotcha :+1: